spark-jobs/src/test/resources/data/standardization_csv_suite_data.csv

```diff
@@ -0,0 +1,5 @@
+a:Aria:Stark
```

Comment:
```suggestion
a:Arya:Stark
```

Reply:
Thanks, will stuck to Star Wars characters next time, there I have less issues with spelling (but what if Disney then comes with copyright claims 🤔 ?)